src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface PaginationProps {
+  arrItems: string[];
+  total: number;
+  perPage: number;
+  setCurrentPage: (page: number) => void;
+  currentPage: number;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  arrItems,
+  total,
+  perPage,
+  setCurrentPage,
+  currentPage,
+}) => {
+  const pageNumbers: number[] = [];
+
+  for (let i = 1; i <= Math.ceil(total / perPage); i++) {
+    pageNumbers.push(i);
+  }
```

Comment:
The pageNumbers array is being manually populated using a for loop. Consider using the `getNumbers` utility function that you've imported on line 3, which seems to be designed for this purpose. This would make the code cleaner and more consistent.